pkg/testhelpers/okta.go

```diff
@@ -123,6 +124,12 @@ func fetchOktaAccessToken(
 		return "", err
 	}
 
+	// Request can return 200 even if session token unset.
+	if authn.SessionToken == "" {
```

Comment:
We might want to use `Status` instead of `SessionToken`. I imagine that's what's dictating failure to authenticate. It would also give us information about why it failed.